store/storetest/preference_store.go

```diff
@@ -192,17 +184,16 @@ func testPreferenceGetAll(t *testing.T, ss store.Store) {
 	err := ss.Preference().Save(&preferences)
 	require.Nil(t, err)
 
-	if result, err := ss.Preference().GetAll(userId); err != nil {
-		t.Fatal(err)
-	} else if len(result) != 3 {
-		t.Fatal("got the wrong number of preferences")
-	} else {
-		for i := 0; i < 3; i++ {
-			if result[0] != preferences[i] && result[1] != preferences[i] && result[2] != preferences[i] {
-				t.Fatal("got incorrect preferences")
-			}
-		}
+	result, err := ss.Preference().GetAll(userId)
+	require.Nil(t, err)
+	require.Equal(t, 3, len(result), "got the wrong number of preferences")
+
+	for i := 0; i < 3; i++ {
+		require.Equal(t, result[0], preferences[i], "got incorrect preferences")
+		require.Equal(t, result[1], preferences[i], "got incorrect preferences")
+		require.Equal(t, result[2], preferences[i], "got incorrect preferences")
```

Comment:
Failing on this test since the returned results should match user preferences.  It should be:
```go
assert.Falsef(t, result[0] != preferences[i] && result[1] != preferences[i] && result[2] != preferences[i], "got incorrect preferences")
```